manticore/core/cpu/disasm.py

```diff
@@ -141,6 +138,18 @@ def unimplemented(self, il):
         return (il.operation == enums.LowLevelILOperation.LLIL_UNIMPL or
                 il.operation == enums.LowLevelILOperation.LLIL_UNIMPL_MEM)
 
+    def _get_current_llil_func(self, code, pc):
+        if pc in self.llil_func_cache:
```

Comment:
Hm I see. So right now, we only cache the Functions and LLILFunctions, and not the instructions. This is again a problem right?